### PR TITLE
Add check-url script

### PR DESCRIPTION
**Pull request by glepretre**
Could you tell me what you think @gentooboontoo? :relaxed:

### CODE REVIEW

File: check-url [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+URL="$1"
+
+if [[ $URL == ^http://* ]]; then
```
Comment (gentooboontoo) [inline check-url]: Not sure it really works

File: check-url [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  URL="http://$URL"
+fi;
+
+while true; do
+  STATUS=$( curl -w '%{http_code}' -s --output /dev/null "$URL")
```
Comment (gentooboontoo) [inline check-url]: I would test the exit status (`$?`) of `curl` by using the `--fail` flag instead.

File: check-url (hunk 1 above)
Comment (glepretre) [inline check-url]: It doesn't, I forgot to invert the assertion and it seems useless using `curl`... :disappointed: 

My bad, thanks :wink:

**Comment by glepretre** [comment]
@gentooboontoo Is it tiny and clean enough now? :sweat_smile:

**Pull request merged**